assets/scss/_registry.scss

```diff
@@ -39,3 +39,47 @@
 .registry-entry {
   @extend .shadow;
 }
+
+// Optimize styles and responsiveness for dark and light mode, see https://getbootstrap.com/docs/5.3/customize/color-modes/
+#searchForm .btn.btn-outline-success,
+.btn-outline-danger,
+.btn-outline-secondary {
+  &:hover {
+    color: var(--bs-white);
+  }
+}
+
+.badge.rounded-pill.text-bg-primary {
+  font-weight: $font-weight-bold;
+}
```

Comment:
this change has no effect, this might be depending on the font that is used. I suggest we remove it for now

```suggestion
```